sklearn/utils/_scipy_sparse_lsqr_backport.py

```diff
@@ -126,7 +126,8 @@ def lsqr(A, b, damp=0.0, atol=1e-8, btol=1e-8, conlim=1e8,
         Right-hand side vector ``b``.
     damp : float
         Damping coefficient.
-    atol, btol : float
+    atol : float, default 1.0e-8
```

Comment:
No, this doesn't make sense without description. I'd rather `atol, btol` as before.